packages/ethereum-contracts/contracts/interfaces/superfluid/Definitions.sol

```diff
@@ -212,6 +230,41 @@ library BatchOperation {
      * )
      */
     uint32 constant internal OPERATION_TYPE_SUPERFLUID_CALL_APP_ACTION = 2 + 200;
+    /**
+     * @dev DMZForwarder.forwardCall batch operation type
+     *
+     * Call spec:
+     * forwardCall(
+     *     target,
+     *     data
+     * )
+     */
+    uint32 constant internal OPERATION_TYPE_SIMPLE_FORWARD_CALL = 1 + 300;
+    /**
+     * @dev DMZForwarder.forward2771Call batch operation type
+     *
+     * Call spec:
+     * forward2771Call(
```

Comment:
to be in sync with the rest, use typed signature, and call "data" "callData" to be more specific.